Documentation/Helm-Charts/operator-chart.md

```diff
@@ -122,6 +122,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.serviceMonitor.enabled` | Enable ServiceMonitor for Ceph CSI drivers | `false` |
 | `csi.serviceMonitor.interval` | Service monitor scrape interval | `"5s"` |
 | `csi.serviceMonitor.labels` | ServiceMonitor additional labels | `{}` |
+| `csi.serviceMonitor.namespace` | ServiceMonitor alternative namespace | `nil` |
```

Comment:
This change is also needed in operator-chart.gotmpl.md, since this doc is generated.

Reply:
This is now generated from the `make helm-docs` target rather than edited by hand